redisless/storage/src/lib.rs

```diff
@@ -2,6 +2,7 @@ pub mod in_memory;
 
 pub trait Storage {
```

Comment:
I think we should replace `set(..)`, `get(..)`, etc.. operations by `write(..)`, `read(..)`, `delete(..)` operations that are closer to disk/storage operations than Redis commands. What do you think @pjeziorowski ?